wagtail/admin/views/pages/listing.py

```diff
@@ -280,20 +280,22 @@ def order_queryset(self, queryset):
         if self.ordering == "ord":
             # preserve the native ordering from get_children()
             pass
-        elif self.ordering == "latest_revision_created_at":
+        elif self.ordering == "latest_revision_created_at" and not self.is_searching:
```

Comment:
Instead of adding `is_searching` checks here, should we do it in `get_ordering`, like we do for `content_type`? Also, I noticed the same error occurs when ordering search results by `content_type`, since the `is_searching` check in there now also has check for `is_explicitly_ordered` (which, looking at it again, I'm not sure is needed).

I also noticed this view class doesn't override `get_valid_orderings` and instead put the logic in `get_ordering` directly, not sure if that was deliberate or not.

Would be nice to remove the sorting link from the column header altogether for these cases, but it may not be straightforward...

Reply:
That's fair enough! I actually added `get_valid_orderings` back to this view as part of #11485, so it shouldn't be too hard to reshuffle things a bit between that PR and this one.

Reply:
> Would be nice to remove the sorting link from the column header altogether for these cases, but it may not be straightforward...

I wonder if making `columns` a `cached_property` in this case would make sense, as it also allows us to put the logic for swapping bulk actions with `OrderingColumn` in the same place. Then `sort_key` can be applied dynamically to turn off the sorting when searching.

However, it does mean the `columns` are now defined for the view instance rather than the class, which wouldn't play well for #11485.

Reply:
Actually, now I look closer, `latest_revision_created_at` _should_ be handled in `order_queryset`, not `get_ordering` / `get_valid_ordering`, because we're not removing it as a valid ordering for search results - just skipping the special case for handling nulls and falling through to the `super().order_queryset` which will just order it as an ordinary field.

I will update this to bring in https://github.com/wagtail/wagtail/pull/11485/commits/fc1d4b309a5628633f2ea5a73220e72e7d2cd857, though, as that contains the fix for `content_type`.